cfme/tests/cli/test_appliance_console.py

```diff
@@ -897,8 +899,35 @@ def test_appliance_ssl():
         casecomponent: Appliance
         caseimportance: medium
         initialEstimate: 1/3h
+        testSteps:
+            1. 'cat /opt/rh/cfme-appliance/COPY/etc/manageiq/postgresql.conf.d/
+            01_miq_overrides.conf' run above command and grep to "ssl = on" string
+            2. run "cat /opt/rh/cfme-appliance/TEMPLATE/var/lib/pgsql/data/pg_hba.conf" on version
+             5.11 and "cat /opt/rh/cfme-appliance/TEMPLATE/var/opt/rh/rh-postgresql95/lib/pgsql/
+             data/pg_hba.conf" on 5.10 version
+        expectedResults:
+            1. confirm "ssl = on" text in 01_miq_overrides.conf file
+            2. confirm "hostssl all         all   all     md5" text in pg_hba.conf file
     """
-    pass
+    command = (
+        "cat /opt/rh/cfme-appliance/COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides.conf"
+    )
+    result = appliance.ssh_client.run_command(command)
+    assert result.success, "SSL check command failed"
+    assert "ssl = on" in result.output, "ssl entry not found"
+
+    command = VersionPick(
+        {
+            LOWEST: (
+                "cat /opt/rh/cfme-appliance/TEMPLATE/var/opt/rh/rh-postgresql95/lib/pgsql/data/"
```

Comment:
TODO: capture these in `cfme.utils.appliance.db`

The `pg_prefix` property captures part of it (`opt/rh/rh-postgresql95` injected path in later versions), but it would be good to have the data directory accessible consistently through a property for use in tests.